flake.nix

```diff
@@ -1,66 +1,76 @@
 {
   description = "Swayfx development environment";
-  inputs.nixpkgs.url = "github:nixos/nixpkgs/nixpkgs-unstable";
+  inputs = {
+    nixpkgs.url = "github:nixos/nixpkgs/nixpkgs-unstable";
+    scenefx.url = "github:ozwaldorf/scenefx";
```

Comment:
this should point to the wlrfx/scenefx repo (I'd assume once your flake there is merged)